src/components/site/Layout.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import Nouislider from 'nouislider-react';
+import API from './API';
+import Chart from './Chart';
+import 'nouislider/distribute/nouislider.css';
+import './StyleSlider.css';
+
+class Practice extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: {},
+      sliderVal1: 0,
+      sliderVal2: 0,
+    };
+  }
+
+  componentDidMount() {
+    API.get('static-data/SDSS_SpecGals_DR8_25000.json').then(res => {
+      console.log(res.data);
+      this.setState(prevState => ({
+        ...prevState,
+        data: res.data,
+      }));
+    });
+  }
+
+  onUpdate = () => value => {
```

Comment:
see my feedback on line 81, implement it, and then this should be `onUpdate = value => {`